### PR TITLE
Support for Django >=4.2

### PR DESCRIPTION
**Pull request by fellu**

### CODE REVIEW

File: s3manifestcollectstatic/management/commands/s3manifestcollectstatic.py [hunk 1]
```diff
@@ -33,17 +34,25 @@ def handle(self, *args, **options):
 
         with TemporaryDirectory() as tmpdirname:
             saved_static_root = settings.STATIC_ROOT
-            saved_storage = settings.STATICFILES_STORAGE
+
+            if VERSION[0] >= 4 and VERSION[1] >= 2:
```
Comment (Tupeco) [inline s3manifestcollectstatic/management/commands/s3manifestcollectstatic.py:38]: `STORAGES` is not yet mandatory in Django 4.2 (just mutually exclusive with the old `STATICFILES_STORAGE`) so these might be better served with some getattrs.

**Review by Tupeco — CHANGES_REQUESTED** [review]
STORAGES is not yet mandatory in Django 4.2 (just mutually exclusive with the old STATICFILES_STORAGE) so these might be better served with some getattrs.

**Comment by dduong42** [comment]
Thank you @fellu and @Tupeco. I've changed this commit a little bit and released a new version.

**Comment by fellu** [comment]
> Thank you @fellu and @Tupeco. I've changed this commit a little bit and released a new version.

thanks! And also in general thanks for keeping this alive, pretty crazy that this kind of functionality is not part of any storage solution :F

**Pull request closed without merging**